Ginger/GingerCore/Environments/Database.cs

```diff
@@ -255,61 +264,57 @@ public string GetConnectionString()
             bool res;
             res = false;
 
-            if (String.IsNullOrEmpty(ConnectionStringCalculated) == false)
+            if (String.IsNullOrEmpty(ConnectionStringCalculated))
             {
-                connStr = ConnectionStringCalculated.Replace("{USER}", UserCalculated);
-     
-                String deCryptValue = EncryptionHandler.DecryptString(PassCalculated, ref res, false);
-                if (res == true)
-                    { connStr = connStr.Replace("{PASS}", deCryptValue); }
-                else
-                    { connStr = connStr.Replace("{PASS}", PassCalculated); }
+                connStr = CreateConnectionString();
             }
+            connStr = ConnectionStringCalculated.Replace("{USER}", UserCalculated);
+            String deCryptValue = EncryptionHandler.DecryptString(PassCalculated, ref res, false);
+            if (res == true)
+            { connStr = connStr.Replace("{PASS}", deCryptValue); }
             else
-            {
-                String strConnString = TNSCalculated;
-                String strProvider;
-                connStr = "Data Source=" + TNSCalculated + ";User Id=" + UserCalculated + ";";
+            { connStr = connStr.Replace("{PASS}", PassCalculated); }
+            
+            return connStr;
+        }
 
-                String deCryptValue = EncryptionHandler.DecryptString(PassCalculated, ref res, false);
-                //ConnectionString = connStr + "Password=" + Pass + ";";
-                if (res == true) { connStr = connStr + "Password=" + deCryptValue + ";"; }
-                else { connStr = connStr + "Password=" + PassCalculated + ";"; }
+        public string CreateConnectionString()
+        {
+            String strProvider;
+            ConnectionString = "Data Source=" + TNS + ";User Id={USER};Password={PASS};";
 
-                if (DBType == eDBTypes.MSAccess)
-                {
-                    if (strConnString.Contains(".accdb")) strProvider = "Provider=Microsoft.ACE.OLEDB.12.0;";
-                    else strProvider = "Provider=Microsoft.Jet.OLEDB.4.0;";
+            if (DBType == eDBTypes.MSAccess)
+            {
+                if (TNSCalculated.Contains(".accdb")) strProvider = "Provider=Microsoft.ACE.OLEDB.12.0;";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5217045)